panel/models/vizzu.py

```diff
@@ -34,6 +35,10 @@ class VizzuChart(LayoutDOM):
         f"{config.npm_cdn}/vizzu@{VIZZU_VERSION}/dist/vizzu.min.js"
     ]
 
+    @classproperty
+    def __javascript__(cls):
```

Comment:
Don't think this is correct. Let's skip Vizzu for now. We'll likely move it into an extension anyway.